docs/api/_index.md

```diff
@@ -663,6 +673,13 @@ GET /status
 
 Displays a web page with the current status of the Alertmanager, including the Alertmanager cluster members.
 
+### Alertmanager configs
+
+```
+GET /multitenant_alertmanager/configs
+```
+
+List all Alertmanager configurations that are currently loaded. This endpoint returns a YAML dictionary with all the Alertmanager configurations and `200` status code on success.
```

Comment:
Same comment about "loaded"... all AM configs are returned. Once AM sharding is finished, AM will only have subset of configs loaded in memory, but endpoint will still return all configs.

I'd also add mention that this endpoint is available regardless of whether alertmanager-API is enabled or not.